### PR TITLE
Update to use cases and temporalVs sections

### PR DESCRIPTION
**Pull request by tsurdilo**
Signed-off-by: Tihomir Surdilovic <tihomir@temporal.io>

## What was changed:

* Update to the use cases section
* Update to the TemporalVs section
## Why?

I updated the two sections to better elaborate on the upsides of using temporal.
Also fixed some minor spelling issues
## Checklist

1. Closes issue: 

2. How was this tested:

3. Any docs updates needed?

### CODE REVIEW

File: components/UseCases.js [hunk 1]
```diff
@@ -35,34 +35,35 @@ export default function UseCases() {
...
           <ExternalLinkWithSVGIcon url="https://docs.temporal.io/docs/use-cases-orchestration/#next-steps">
             Examples from Uber and Banzai Cloud
           </ExternalLinkWithSVGIcon>
         </UCaseItem>
         <UCaseItem header="Financial Transactions">
           <h2 className="font-bold mb-3">
-            For software that interacts with money (especially someone else's), reliability and
-            consistency is the highest priority.
+            For software that interact with money (especially someone else's), reliability and
+            consistency are top priorities.
```
Comment (flossypurse) [inline components/UseCases.js:61]: I like this change.

File: components/UseCases.js [hunk 2]
```diff
@@ -75,8 +76,8 @@ export default function UseCases() {
               logic
             </li>
             <li className="mb-2">Support for long-running operations</li>
-            <li className="mb-2">Guarantees that workflow code eventually completes</li>
-            <li className="mb-2">Has built-in support for exponential activity retries</li>
+            <li className="mb-2">Completion guarantee for workflows of any size and complexity</li>
+            <li className="mb-2">Built-in support for exponential activity retries</li>
```
Comment (flossypurse) [inline components/UseCases.js:80]: Good catch on list consistency.

File: components/UseCases.js [hunk 3]
```diff
@@ -96,8 +97,9 @@ export default function UseCases() {
             require a custom provisioning flow.
           </p>
           <p className="mb-3">
-            Some provisioning operations may take dozens of minutes or even hours to complete.
-            Ad-hoc solutions may fail in the middle and leave the system in an undefined state.
+            Some provisioning operations may take hours or event days to complete. Ad-hoc
```
Comment (flossypurse) [inline components/UseCases.js:100]:
```suggestion
            Some provisioning operations may take hours or even days to complete. Ad-hoc
```

File: components/TemporalVs.js [hunk 4]
```diff
@@ -21,60 +21,76 @@ export default function TemporalVs() {
...
-            framework. Temporal enables the development of apps of all shapes, sizes and scales.
+            While Temporal is a great option for data pipelines, this encompasses only a very small
+            subset of its use cases. Temporal enables development of applications of all shapes and
+            sizes, regardless of scale.
           </p>
           <p className="mb-3">
-            Temporal targets developers and all code written with Temporal runs directly instead of
-            compiling a intermediate DAG. This gives you flexibility which is impossible to achieve
-            with more rigid DAG based systems.
+            Temporal targets developers and all code written with Temporal runs directly,
```
Comment (flossypurse) [inline components/TemporalVs.js:47]:
```suggestion
            Temporal targets developers. All code written with Temporal runs directly,
```

File: components/TemporalVs.js [hunk 5]
```diff
@@ -21,60 +21,76 @@ export default function TemporalVs() {
...
           <p className="mb-3">
-            Temporal provides a strongly-consistent code-first development environment which can be
-            used for orchestrating microservices, building data pipelines, provisioning resources
-            and much more. Code written with Temporal is executed directly which enables users to
-            use the development, debugging and testing processes they already know and love.{' '}
+            Flow-chart based workflow DSLs suffer from being bound to a specific set of features and
+            capabilities provided by the underlying workflow language they use. Even though they
+            promote the visualization aspect of workflow design, the actual execution semantics are
+            expressed as a mix of programming and expression languages which are enforced by the
+            specific runtime implementation. Temporal does not suffer from these type of
```
Comment (flossypurse) [inline components/TemporalVs.js:63]:
```suggestion
            specific runtime implementation. Temporal does not suffer from these types of
```

File: components/TemporalVs.js [hunk 6]
```diff
@@ -21,60 +21,76 @@ export default function TemporalVs() {
...
+            restrictions allowing you to fully define your workflows using a programming language.
```
Comment (flossypurse) [inline components/TemporalVs.js:64]:
```suggestion
            restrictions enabling you to fully define your workflows using a programming language.
```

File: components/TemporalVs.js [hunk 7]
```diff
@@ -21,60 +21,76 @@ export default function TemporalVs() {
...
+            This provides a high level of flexibility not only with defining your workflow logic but
+            also testing and debugging in the development environment of your choice.
           </p>
           <p className="mb-3">
-            Temporal targets developers and all code written with Temporal runs directly instead of
-            being translated to an intermediate representation. This gives you flexibility which is
-            impossible to achieve with DSL approaches.
+            BPM engines translate the workflow definitions markup into code, which requires
```
Comment (flossypurse) [inline components/TemporalVs.js:69]:
```suggestion
            BPM engines translate workflow definition markup into code, which requires
```

File: components/TemporalVs.js [hunk 8]
```diff
@@ -21,60 +21,76 @@ export default function TemporalVs() {
...
+            additional compilation steps. This code generation is done differently by each runtime
```
Comment (flossypurse) [inline components/TemporalVs.js:70]:
```suggestion
            additional compilation steps. This kind of code generation is approached differently by each runtime
```

File: components/TemporalVs.js [hunk 9]
```diff
@@ -21,60 +21,76 @@ export default function TemporalVs() {
...
+            engine using tools they chose. Temporal eliminates the intermediate compilation steps
+            allowing you to express your workflow logic directly in code. In addition, Temporal
```
Comment (flossypurse) [inline components/TemporalVs.js:72]:
```suggestion
           enabling you to express your workflow logic directly in code. In addition, Temporal
```

File: components/UseCases.js [hunk 10]
```diff
@@ -35,34 +35,35 @@ export default function UseCases() {
...
           </p>
           <ExternalLinkWithSVGIcon url="https://docs.temporal.io/docs/use-cases-orchestration/#next-steps">
             Examples from Uber and Banzai Cloud
           </ExternalLinkWithSVGIcon>
         </UCaseItem>
         <UCaseItem header="Financial Transactions">
           <h2 className="font-bold mb-3">
-            For software that interacts with money (especially someone else's), reliability and
-            consistency is the highest priority.
+            For software that interact with money (especially someone else's), reliability and
```
Comment (flossypurse) [inline components/UseCases.js:60]:
```suggestion
            For software that interacts with money (especially someone else's), reliability and
```

Plural form is correct here.

File: components/UseCases.js [hunk 11]
```diff
@@ -155,8 +158,8 @@ export default function UseCases() {
             parallel cron jobs that don't require a separate batch processing framework.
           </p>
           <p className="mb-3">
-            Temporal support for long-running activities and unlimited retries also makes it a great
-            fit for monitoring use cases.
+            Temporal's support for long-running activities and unlimited retries also makes it a
```
Comment (flossypurse) [inline components/UseCases.js:161]:
```suggestion
            Temporal's support for long-running Activities and unlimited retries also makes it a
```

In this case we want to capitalize Activities.

I may have missed some above, but basically anytime we refer to a Workflow and Activity as a concept/component of the Temporal system - capitalize it.
That way users can differentiate between generic "workflow" terminology and the Temporal system.

**Review by flossypurse — CHANGES_REQUESTED** [review]
Overall, great refresh.
I love the attention to consistency in tone and tenses.

Just a few small typos that you will want to fix first.

**Comment by tsurdilo** [comment]
@flossypurse thanks for the great review! I applied all of your suggestions

**Review by sw-yx — APPROVED** [review]

**Comment by sw-yx** [comment]
nice tweaks

**Pull request merged**